src/main/java/inf112/skeleton/app/object/Card.java

```diff
@@ -0,0 +1,16 @@
+package inf112.skeleton.app.object;
+
+public class Card {
+    private CardType cardType;
+    private int priority;
```

Comment:
Should we have a method/function that gives the card priority also, since we need it at to decide who goes first?
Or maybe this can be a `public final int` (if I understand `final ` correct)
Not critical at this point, or the merge, but should be added at a later date.